### ISSUE TITLE
save with vertex color

### ISSUE THREAD

**Issue by JasonBoy1** [facebookresearch/pytorch3d#834]
hello, seems that pytorch3d cannot save mesh with TexturesVertex. Is that so?
[save_obj](https://github.com/facebookresearch/pytorch3d/blob/main/pytorch3d/io/obj_io.py#L661-L670)

**Comment by JasonBoy1** [comment]
While in obj file, vertex color is supported.

**Comment by patricklabatut** [comment]
See the discussion on [#779](https://github.com/facebookresearch/pytorch3d/issues/779). The issue was closed by the user but is still tracked internally.

**Comment by monniert** [comment]
yes I closed it as I felt it was not a priority, good to here it is on your roadmap at some point, I am reopening it to avoid duplicates then
